src/main/java/competition/subsystems/pose/PoseSubsystem.java

```diff
@@ -80,4 +82,23 @@ private SwerveModulePosition[] getSwerveModulePositions() {
                 drive.getRearRightSwerveModuleSubsystem().getCurrentPosition()
         };
     }
+
+    public void setCurrentPosition(double newXPositionMeters, double newYPositionMeters, WrappedRotation2d heading) {
+        super.setCurrentPosition(newXPositionMeters, newYPositionMeters);
+        super.setCurrentHeading(heading.getDegrees());
+        onlyWheelsGyroSwerveOdometry.resetPosition(
+                heading,
+                getSwerveModulePositions(),
+                new Pose2d(
+                        newXPositionMeters,
+                        newYPositionMeters,
+                        this.getCurrentHeading()));
+    }
+    public void setCurrentPoseInMeters(Pose2d newPoseInMeters) {
```

Comment:
⭐ formatting nit, add newline between functions (this should be failing stylecheck I would have thought)